rllib/models/utils.py

```diff
@@ -153,6 +153,69 @@ def get_activation_fn(
     )
 
 
+@DeveloperAPI
+def get_initializer_fn(name: Optional[Union[str, Callable]], framework: str = "torch"):
+    # Already a callable return as is.
```

Comment:
Could we add a docstring here?

Reply:
Sure, I was looking for an initial review. I will add them in the next commits.

Reply:
Done